classes/webservice/WebserviceRequest.php

```diff
@@ -372,6 +382,8 @@ public static function getResources()
     /**
      * This method is used for calculate the price for products on the output details.
      *
+     * @deprecated Since 1.6.0.10
```

Comment:
I traced back the commit that removed calls to this function: https://github.com/PrestaShop/PrestaShop/commit/13748d360eaee40c71ddd81761ab9dcdb364b9ee